script.js

```diff
@@ -0,0 +1,152 @@
+const brand = document.querySelector('.brand');
+const checkbox = document.querySelector('#toggle');
+const intro = document.querySelector('.intro');
+const hamburger = document.querySelector('.hamburger');
+const concert = document.querySelector('.concert');
+const aboutPast = document.querySelector('.about-past');
+const concertWrap = document.querySelector('.concert-wrap');
+const singerWrap = document.querySelector('.singer-wrapper');
+const mainProgram = document.querySelector('.mainProgram');
+const featureSingers = document.querySelector('.featureSingers');
+const partners = document.querySelector('.partners');
+const footerWrap = document.querySelector('.footer-wrap');
+const detailWrap = document.querySelector('.detail-wrap');
+
+function notEmpty(element) {
+  return element !== null;
+}
+function remove() {
+  document.getElementById('toggle').checked = false;
+  if (notEmpty(intro)) {
+    intro.classList.remove('blur');
+  }
+  if (notEmpty(mainProgram)) {
+    mainProgram.classList.remove('blur');
+  }
+  if (notEmpty(featureSingers)) {
+    featureSingers.classList.remove('blur');
+  }
+
+  if (notEmpty(partners)) {
+    partners.classList.remove('blur');
+  }
+  if (notEmpty(footerWrap)) {
+    footerWrap.classList.remove('blur');
+  }
+  if (notEmpty(detailWrap)) {
+    detailWrap.classList.remove('blur');
+  }
+
+  if (notEmpty(concert)) {
+    concert.classList.remove('blur');
+  }
+  if (notEmpty(aboutPast)) {
+    aboutPast.classList.remove('blur');
+  }
+  if (notEmpty(concertWrap)) {
+    concertWrap.classList.remove('blur');
+  }
```

Comment:
- [ ]  Kindy refactor this code to be less repetitive because it does not follow the DRY best practice of JavaScript, You can use something like the following:
`const allElements = documents.querySelectorAll('class1, class2, class3....')`

and then inside your remove function, you can use something like this:

`allElements.forEach(element => { if (notEmpty(element)) { element.classList.remove('blur') } })`